M2/Macaulay2/packages/GKMManifolds/Documentation_GKMManifolds.m2

```diff
@@ -0,0 +1,2165 @@
+		C:TKClass
+		n:ZZ
+	Outputs
+		:TKClass
+			the n-th power of C
+	Description
+		Text
+			This method computes the $n$-th power of a $T$-equivariant $K$-class $C$.
+
+		Example
+			Gr24 = tGeneralizedFlagVariety("A",3,{2}); --the Grassmannian of projective lines in projective 3-space
+			O1 = ampleTKClass Gr24 -- the O(1) bundle on Gr24 as a T-equivariant K-class
+			O2 = O1^2
+			peek O2
+			Oneg1 = O1^(-1)
+			peek Oneg1
+
+	Caveat
+		$n$ is allowed to be negative only when $C$ is a line bundle, or a direct sum of copies of a line bundle.
+
+	SeeAlso
+		tKClass
+		(symbol *, TKClass, TKClass)
+		(symbol +, TKClass, TKClass)
+///
+
+
+doc ///
+	Key
+		(symbol +, TKClass, TKClass)
+	Headline
+		computes the sum of two T-equivariant K-classes
+	Usage
+		C1 + C2
+	Inputs
+		C1:TKClass
+		C2:TKClass
+	Outputs
+		:TKClass
+			the sum of C1 and C2
+	Description
+		Text
+			This method computes the sum of two $T$-equivariant $K$-classes.
+
+		Example
+			Gr24 = tGeneralizedFlagVariety("A",3,{2}); --the Grassmannian of projective lines in projective 3-space
+			O1 = ampleTKClass Gr24 -- the O(1) bundle on Gr24 as a T-equivariant K-class
+			E = O1 + (O1*O1)
+			peek E
+
+	SeeAlso
+		tKClass
+		(symbol *, TKClass, TKClass)
+///
+
+doc ///
+	Key
+		tGeneralizedFlagVariety
+		(tGeneralizedFlagVariety, String, ZZ, List)
+		(tGeneralizedFlagVariety, String, ZZ, List, Ring)
+	Headline
+		makes a generalized flag variety as a T-variety
+	Usage
+		X = tGeneralizedFlagVariety(LT,d,L)
+		X = tGeneralizedFlagVariety(LT,d,L,R)
+	Inputs
+		LT:String
+			one of "A", "B", "C", or "D"
+		d:ZZ
+			the dimension of the root system
+		L:List
+			of integers strictly between 1 and d (inclusive)
+		R:Ring
+			the character ring of the torus acting on the generalized flag variety
+
+	Outputs
+		X:TVariety
+			representing the corresponding generalized flag variety
+
+	Description
+		Text
+			Let $G$ be the Lie group corresponding to $LT_d$, and
+			let $w = a_1w_1 + \cdots + a_dw_d$ be a nonnegative $\mathbb Z$-linear combination of fundamental weights 
+			in the root system of type $LT_d$, where $a_i$ is the number of times $i$ appears in the list $L$.
+			(See @TO "Example: generalized flag varieties"@ for conventions regarding classical Lie groups and 
+			their root systems).
+			This method outputs the T-variety representing the generalized flag variety $G/P$ embedded in the irreducible 
+			representation of $G$ with the highest weight $w$.
+
+		Text
+			The following example features the Lagrangian Grassmannian $LGr(2,4)$ of 2-dimensional subspaces
+			in $\mathbb C^4$ that are isotropic under the standard alternating form.  Its @TO MomentGraph@ is a complete 
+			graph on 4 vertices.
+
+		Example
+			LGr24 = tGeneralizedFlagVariety("C",2,{2})
+			peek LGr24
+			momentGraph LGr24
+			tChi ampleTKClass LGr24
+
+	Caveat
+		Spin groups have not been implemented.
+
+	SeeAlso
+		"Example: generalized flag varieties"
+		tFlagMap
+
+
+///
+
+
+
+doc ///
+	Key
+		TMap
```

Comment:
Maybe better is "GKMMap", leaving equivariance implicit, since "T" is meaningless.